### DISCUSSION TITLE
Add support for S/MIME protocol

### DISCUSSION THREAD

**Discussion by tatticoder** [simple-login/app#535] [Feature Requests]
I primarily use simple login to send or receive emails from my domain based email to my personal gmail inbox. Is it possible for me to somehow attach S/MIME certificate generated for my domain based email address to outgoing emails.

As mentioned in #527 I tried to send email using Mozilla Thunderbird on Fedora Linux but since sender email (`abc@gmail.com`) differs from email in certificate (`abc@example.com`) it does not allow me to attach that. 

I know most people will be using simple login to hide their primary email address from getting spammed and this feature will be useless to them but it would still be nice to have for professional users who want additional layer of security and verification in their communication channels.

**Comment by muibusan** [comment]
Thumbs up because S/MIME is a fully built-in feature in iOS and would allow to read encrypted mails easily in iOS devices, too. OpenPGP-encrypted emails on the other hand can hardly be dealt with on iOS devices.

**Reply by gitrahab** [reply]
This would be a great addition. Rumor is that AnonAddy is looking to add this feature. Hope to get it here in SimpleLogin as my preferred alias system (and premium member)

**Comment by Derad6709** [comment]
Any updates?

**Reply by aisknab** [reply]
Looking for this as well

**Comment by matchcom-cahors** [comment]
It's a bad thing that there have been no updates on this yet. 😞  
S/MIME 🔏 would be important, especially for use on iOS. 📱

**Comment by schwigri** [comment]
I made a PR here: https://github.com/simple-login/app/pull/1946

Feel free to clone this into your own self-hosted instance as well as help expand on the PR if you would like :) I am using it myself currently and it makes integration with iOS and macOS really nice!
